perun-base/src/main/java/cz/metacentrum/perun/core/api/CandidateGroup.java

```diff
@@ -0,0 +1,79 @@
+package cz.metacentrum.perun.core.api;
+
+/**
+ * @date 8/30/17.
+ * @author Peter Balcirak peter.balcirak@gmail.com
+ */
+public class CandidateGroup extends Group {
```

Comment:
If it is possible, it would be better to use composition instead of inheritance. Currently other Beans in Perun are implemented this way, but we would like to get rid of it in the future since it is causing a lot of problems (e.g. implementation of equals and hashCode methods).

Reply:
Done